bewcloud.config.sample.ts

```diff
@@ -18,6 +18,7 @@ const config: PartialDeep<Config> = {
   // files: {
   //   rootPath: 'data-files',
   //   allowPublicSharing: false, // If true, public file sharing will be allowed (still requires a user to enable sharing for a given file or directory)
+  //   allowDirectoryDownloads: true, // If true, directories can be downloaded as zip files
```

Comment:
Let's default this to `false` as it'll require some resources and take up UI space on hover.